packages/@core.next/auth.jsx

```diff
@@ -25,6 +25,8 @@ const userFragment = `
   avatar {
     publicUrl
   }
+  phone
```

Comment:
You don't have access to this fields for all users. Probably it should fail tests. 

You can create a custom GQL query for your profile. It's better to discuss it

Reply:
https://doma.atlassian.net/browse/SBERDOMA-357

create task for this concern